packages/go_router/lib/go_router.dart

```diff
@@ -11,7 +11,7 @@ export 'src/configuration.dart'
 export 'src/misc/extensions.dart';
 export 'src/misc/inherited_router.dart';
 export 'src/pages/custom_transition_page.dart';
-export 'src/route_data.dart' show GoRouteData, TypedGoRoute;
+export 'src/route_data.dart' show GoRouteData, TypedGoRoute, TypedShellRoute;
```

Comment:
@hawkkiller Do we also need to export `ShellRouteData` here?

Reply:
As you can see it is already exported. 

If the question lies in the export itself then it is needed because of visibility purposes. Just create a fork and check the behaviour without it :) you won't be able to use this class.

Reply:
I am running the latest code from this branch and while I can use `TypedShellRoute` I can not use the `ShellRouteData` class because it is not found. This is my question.

Reply:
How can `ShellRouteData` be exported if it's not included here? I think we need to add it.